code/game/objects/structures/gym/weight_machine.dm

```diff
@@ -18,6 +18,12 @@
 	///The weight action we give to people that buckle themselves to us.
 	var/datum/action/push_weights/weight_action
 
+	///the most drunk you can be to safely use the machine
+	var/safe_drunk_level = 39
```

Comment:
It's only used in one place and not overriden elsewhere. You may as well make it a define (undefined at the end of the file) instead.